src/langcheck/metrics/eval_clients/_base.py

```diff
@@ -222,3 +225,35 @@ def compute_metric_values_from_template(
             metric_values=scores,
             language=language,
         )
+
+    def augment_from_template(
+        self,
+        instances: list[str] | str,
+        template: Template,
+        num_perturbations: int = 1,
+    ) -> list[str | None]:
+        """Augments the instances using the given Jinja template for
+        `num_perturbations` times.
```

Comment:
I think it is worth noting here that every EvalClient subclass needs to be implemented so that calling `get_text_responses` with different prompts will return different results. 

(As far as I checked the `AnthropicEvalClient` returns the same result again and again. I think it's fine, but it would be better to write that this kind of things could happen.)